sql/src/main/java/io/crate/execution/dsl/projection/WindowAggProjection.java

```diff
@@ -63,6 +72,8 @@ public WindowAggProjection(WindowDefinition windowDefinition,
 
     public WindowAggProjection(StreamInput in) throws IOException {
         windowDefinition = new WindowDefinition(in);
+        startFrameOffset = in.readGenericValue();
```

Comment:
Could we use a typed streamer based on the windowDefinition symbols/value type? Not sure if it makes sense / is worth it.